configs/prow-dev/config/external_plugins_config.yaml

```diff
@@ -60,13 +60,10 @@ ti-community-blunderbuss:
       - ti-community-infra/test-dev
     pull_owners_endpoint: https://prow-dev.tidb.io/ti-community-owners
     max_request_count: 2
-    require_sig_label: true
-    exclude_reviewers:
-      # Bots
+    require_sig_label: false
+    include_reviewers:
```

Comment:
Why use robots as reviewers?

Reply:
For test. I don't want to bother you on test-dev repo.

Reply:
OK